README.rst

```diff
@@ -27,15 +27,29 @@ And an Array can be converted to a numpy ndarray on demand::
     array([ 0.72151309,  0.54654914], dtype=float32)
 
 
-Get in touch!
--------------
+------------------------------------------
+Important Note : Future development status
```

Comment:
> within a note or warning admonition

I would prefer that (like our Sphinx ".. note:" sections), but I think it is not achievable with RST
I'm thinking this is the best I can do, unless you have a different suggestion?